indra/cmake/Python.cmake

```diff
@@ -40,7 +40,7 @@ elseif (WINDOWS)
     ${regpaths}
     ${pymaybe}
     )
-    include(FindPythonInterp)
+  find_package(Python3 COMPONENTS Interpreter)
```

Comment:
Hmm! I'm sure this change addresses the deprecation warning, but now I find myself wondering: why did we have this `include(FindPythonInterp)` line at all?

[The CMake FindPythonInterp documentation](https://cmake.org/cmake/help/latest/module/FindPythonInterp.html) says that `FindPythonInterp` sets `PYTHONINTERP_FOUND` and `PYTHON_EXECUTABLE` (along with `PYTHON_VERSION_*`, which we don't seem to reference).

Maybe all we need is to delete line 43 altogether and move lines 47-49 before line 52, turning 52 into `else` ?

Reply:
This is a contribution and it works so I'm taking it as is, we can improve it later.